src/components/map/MapService.js

```diff
@@ -1214,14 +1265,12 @@ goog.require('ga_urlutils_service');
         // or permalink
         // @param olLayerOrId  An ol layer or an id of a layer
         isExternalWmsLayer: function(olLayerOrId) {
-          if (!olLayerOrId) {
-            return false;
-          }
           if (angular.isString(olLayerOrId)) {
             return /^WMS\|\|/.test(olLayerOrId) &&
                 olLayerOrId.split('||').length >= 4;
+          } else {
+            return this.isWMSLayer(olLayerOrId);
```

Comment:
Okay, yes but I haven't really changed anything here. I'll add a test and adapt to check if the layer is a bod layer.
I can't make gaMapUtils depend on gaLayers though as it would create a circular dependency.